superbench/runner/runner.py

```diff
@@ -68,7 +69,12 @@ def __get_mode_command(self, mode, exec_command):
         Return:
             str: Runner command.
         """
-        if mode.name == 'torch.distributed':
+        if mode.name == 'local':
+            return '{prefix} {command}'.format(
+                prefix=(mode.prefix or '').format(proc_rank=mode.proc_rank, proc_num=mode.proc_num or 1),
```

Comment:
Shall we move this logic before return? I think it's hard to debug if there is failure.

Reply:
updated

Reply:
Why prefix need to set `proc_num=mode.proc_num or 1`?